pinax/stripe/tests/test_actions.py

```diff
@@ -961,18 +966,36 @@ def test_subscription_create_token(self, SubscriptionCreateMock, CustomerMock):
         _, kwargs = SubscriptionCreateMock.call_args
         self.assertEquals(kwargs["source"], "token")
 
-    @patch("pinax.stripe.actions.subscriptions.sync_subscription_from_stripe_data")
     @patch("stripe.Subscription.create")
-    def test_subscription_create_with_connect(self, SubscriptionCreateMock, SyncMock):
-        subscriptions.create(self.customer, "the-plan", stripe_account=self.account)
-        self.assertTrue(SyncMock.called)
+    def test_subscription_create_with_connect(self, SubscriptionCreateMock):
+        self.customer.stripe_account = self.account
```

Comment:
Will change the customer for following tests..

Reply:
I suggest using a connected_customer for this.

Reply:
thx, even better